src/core/ucc_constructor.c

```diff
@@ -111,28 +62,51 @@ static ucc_status_t ucc_check_config_file(void)
     return status;
 }
 
+static ucc_status_t init_lib_paths(void)
+{
+    const char  *so_path = ucc_sys_get_lib_path();
+    ucc_status_t status;
+    char        *lib_path;
+
+    if (!so_path) {
+        return UCC_ERR_NOT_FOUND;
+    }
+    lib_path = ucc_sys_dirname(so_path);
+    if (!lib_path) {
+        return UCC_ERR_NOT_FOUND;
+    }
+
+    ucc_global_config.install_path = ucc_sys_dirname(lib_path);
+    if (!ucc_global_config.install_path) {
+        status = UCC_ERR_NO_MEMORY;
+        goto out;
+    }
+    status = ucc_str_concat(lib_path, "/ucc",
```

Comment:
Unify module path from config.h
```suggestion
    status = ucc_str_concat(lib_path, UCC_MODULE_SUBDIR,
```